desktop/nested-virtualization.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Instructions on how to enable nested virtualization
+keywords: nested virtualization, Docker Desktop
+title: Enable nested virtualization
+---
+>Note 
+>
+>This is work in progress. The steps haven’t been validated yet in our test environment.
```

Comment:
@aevesdocker just checking, is this the note you want published?

Reply:
Yes, thanks for checking. It's confirmed with the PM and engineering team